pages/index.vue

```diff
@@ -568,14 +568,13 @@
             </div>
           </div>
           <div id="response-details-wrapper">
-            <pre>
-              <code
-  ref="responseBody"
-  id="body"
-  rows="16"
-  placeholder="(waiting to send request)"
->{{response.body}}</code>
-            </pre>
+            <ResponseBody
+              :value="responseBodyText"
+              theme="dracula"
+              lang="json"
```

Comment:
What if the response body is not JSON?

Reply:
the ace editor `setValue` method just gets _String_, so regardless of response type before passing the value to `ResponseBody` we convert it to _String_ .
the `lang`  determines the highlight syntax and how to show the passed string value.
the `lang` also check syntax and show error if it's wrong(but in response body I disabled it because it's just a readonly api response)

Reply:
@NBTX I made `lang` prop dynamic, changing by response type.